core/cap-0033.md

```diff
@@ -68,8 +68,14 @@ typedef AccountID* SponsorshipDescriptor;
 
 struct AccountEntryExtensionV2
 {
-    uint32 numSponsored;
-    uint32 numSponsoring;
+    uint32 numSponsored;  // Number of reserves sponsored by this account
+                          // This reduces the base reserve requirement
+    uint32 numSponsoring; // Number of reserves sponsored for this account
+                          // This increases the base reserve requirement
```

Comment:
In our working group we said
>numSponsoring: how many (reserves) I’m sponsoring (plus sign in the base reserve formula)
numSponsored:  how many of my reserves are sponsored by someone else (minus sign in the base reserve formula)

This seems like the opposite of the comment in this PR? Which is correct?

Reply:
The comments from the working group are correct. Thanks, fixed.